ocaml/xapi/context.ml

```diff
@@ -347,13 +346,7 @@ let make_subcontext ~__context ?task_in_database task_name =
     let open Tracing in
     Option.bind __context.tracing (fun parent ->
         let parent = Some parent in
-        let tracer = get_tracer ~name:task_name in
-        match Tracer.start ~tracer ~name:task_name ~parent () with
-        | Ok x ->
-            x
-        | Error e ->
-            R.warn "Failed to start tracing: %s" (Printexc.to_string e) ;
-            None
+        start_tracing_helper (fun span_name -> parent) task_name
```

Comment:
The function, while correct, discards the parameter so the compiler will complain about it. Other ways of expressing this, without upsetting the compiler are: `(Fun.id parent)` or `(fun _ -> parent`